### ISSUE TITLE
Inappropriate behavior of Replace All

### ISSUE THREAD

**Issue by Prateek032** [att/rcloud#2270]
Contents in the cell are not getting replaced even after clicking on ` Replace all `

Steps to reproduce:
-  Invoke `Find & Replace`
-  Enter keyword in `Find` and press next
- Now enter the keyword in `Replace` field
- and then press `Replace all`

The first match is not getting replaced

![image](https://cloud.githubusercontent.com/assets/10941328/18500120/62aa9a7a-7a62-11e6-9b06-aa66622aa23b.png)

**Comment by gordonwoodhull** [comment]
I think we discussed this. Some editors implement Replace All as "replace to end" and some implement it as "replace every single occurrence". I chose the former (probably because it's what emacs does) but I'm open to arguments for the latter.

**Comment by Prateek032** [comment]
But i checked in every editors (eg: Sublime, Notepadd ++, gedit) it replaces all the contents.

**Comment by gordonwoodhull** [comment]
Yep, I'm not surprised if Emacs has uncommon behavior here. I'm open to changing it.
